specs/templates/custom-component.md

```diff
@@ -50,6 +50,7 @@
 - *Methods*
 - *Events*
 - *CSS Classes and CSS Custom Properties that affect the component*
+- *Built-in CSS Properties that affect the component. In particular, how is it sized?*
```

Comment:
Think mentioning a couple specific CSS Properties (like width and height) makes it clearer. Also chose "native" over "built-in" but other ideas could be `Browser-native CSS Properties` or `browser built-in CSS Properties`

```suggestion
- *Response to configuring native CSS Properties that affect the component (height, width, etc.)?*
```